### PR TITLE
Feature/user profile

### PR DESCRIPTION
**Pull request by kwreen**
This is a pull request for @davidberard2 's branch. The work done on it was for issue #10 .

### CODE REVIEW

**Comment by kwreen** [comment]
Updating the user profile requires some more work. But the profile page can be displayed with data from Firebase.

File: app/src/main/java/Fragments/ProfileFragment.java [hunk 1]
```diff
@@ -22,17 +48,105 @@ public static ProfileFragment newInstance() {
...
+        // TODO: Set listener for add_photo ImageButton to call addPhoto()
+        addPhoto_ib.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                editPhoto();
+            }
+        });
+
+        // TODO: Set listener for edit_email, edit_phone_number
+    }
```
Comment (kkhh001) [inline app/src/main/java/Fragments/ProfileFragment.java:122]: Maybe this section just add comments such as User Profile Edit, but code is clean so easy to read!

File: app/src/main/java/Fragments/ProfileFragment.java [hunk 2]
```diff
@@ -22,17 +48,105 @@ public static ProfileFragment newInstance() {
...
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_profile, container, false);
+
+        name_tv = (TextView) view.findViewById(R.id.profile_name);
+        email_tv = (TextView) view.findViewById(R.id.profile_email);
+        phoneNumber_tv = (TextView) view.findViewById(R.id.profile_phone_number);
+        photo_iv = (ImageView) view.findViewById(R.id.profile_photo);
+        addPhoto_ib = (ImageButton) view.findViewById(R.id.profile_add_photo);
+
```
Comment (kkhh001) [inline app/src/main/java/Fragments/ProfileFragment.java:65]: Just wondering: what does _tv, _iv and _ib stand for ?

**Review by kkhh001 — APPROVED** [review]
Just adding a bit more comments to separate sections in the code and everything should be good! Good job 👍

**Review by LHuss — APPROVED** [review]

File: app/src/main/java/Fragments/ProfileFragment.java (hunk 2 above)
Comment (davidberard2) [reply app/src/main/java/Fragments/ProfileFragment.java:65]: textView, imageView and imageButton

**Comment by davidberard2** [comment]
Did you implement the method getDisplayName and getEmail ? Otherwise, I think the code seems right. No conflict and good standards have been use.

**Pull request merged**